app/models/version.rb

```diff
@@ -176,6 +178,16 @@ def platformed?
 
   delegate :reorder_versions, to: :rubygem
 
+  def authored_at
+    return built_at if rely_on_built_at?
+
+    created_at
+  end
+
+  def rely_on_built_at?
+    built_at && seeded? && built_at <= RUBYGEMS_IMPORT_DATE
```

Comment:
nit: this is a bit difficult to understand for me. Maybe something like this would be easier to understand

```ruby
def rely_on_built_at?
  return false unless seeded? # or created_at&.to_date == RUBYGEMS_IMPORT_DATE
  built_at && built_at <= RUBYGEMS_IMPORT_DATE
end
```

Reply:
Good point, I think `seeded?` is unnecessary and only complicates readability
Let me just inline the `RUBYGEMS_IMPORT_DATE` comparison